src/app/OperationalSessionSetup.cpp

```diff
@@ -296,7 +296,18 @@ void OperationalSessionSetup::UpdateDeviceData(const ResolveResult & result)
 
 CHIP_ERROR OperationalSessionSetup::EstablishConnection(const ResolveResult & result)
 {
-    auto & config = result.mrpRemoteConfig;
+    auto config = result.mrpRemoteConfig;
+
+    if (result.isICDOperatingAsLIT)
+    {
+        // When ICD is operating as LIT, we need to adjust the MRP idle retransmission timeout
+        // to accommodate the potentially long sleep periods.
```

Comment:
Isn't the key part that if it's operating as LIT then:

1. The server is not going to advertise an SII at all (so it will be 0).
2. In practice we should only be doing CASE if the server is active, so should be using SAI, not SII, for MRP computation.

as a result, it make sense to sense SII to the SAI value, for MRP purposes.